lib/matplotlib/tests/test_lines.py

```diff
@@ -195,7 +195,8 @@ def test_marker_fill_styles():
     x = np.array([0, 9])
     fig, ax = plt.subplots()
 
-    for j, marker in enumerate(mlines.Line2D.filled_markers):
+    for j, marker in enumerate([
+            m for m in mlines.Line2D.filled_markers if m != "."]):
```

Comment:
It's not really worth regen'ing the image.

Reply:
Please make this clear in a comment so that one can easily understand why it's there and remove the exclusion if the image is regenerated for some other reason.

 Btw that test image changes are expensive is a nuisance. Maybe we can move that topic via paid/contract work?

Reply:
Commented.
The baseline image has already been discussed at length elsewhere, let's not derail this PR with it.